portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
Lets change greeting to something else since you're no longer generating a greeting